src/api/app/views/webui/request/beta_show_tabs/_conversation.html.haml

```diff
@@ -58,7 +58,7 @@
     %hr
 
     - if bs_request.accept_at.present?
-      .alert.alert-warning.p-2.mb-3
```

Comment:
While looking at this code, I noticed that the `mb-3` class does nothing since the default margin bottom for alerts is `1rem`, exactly what `mb-3` sets. So I removed it.